python/tk_multi_publish2/dialog.py

```diff
@@ -1066,13 +1118,8 @@ def do_publish(self):
                 self._progress_handler.push("Running finalizing pass")
 
                 try:
-                    # note: Bugfix SG-4584: Re-acquire the parent pointer as we are iterating.
```

Comment:
This shouldn't be an issue anymore since we're not really traversing the items themselves. Instead, the custom task generators use Qt iterators. Let me know if you see any issues with removing this.